### DISCUSSION TITLE
Pagamento rejeitado ao comprar com usuários de teste - (Resolvido)

### DISCUSSION THREAD

**Discussion by EnderMaldonado** [rodurma/forum-meli#7] [Geral]
Venho tentando fazer compras teste, entre usuários teste, conforme explicado na API e sempre obtenho o mesmo resultado
**"Seu cartão recusou o pagamento"**

![image](https://user-images.githubusercontent.com/52002044/144987777-76938e26-ea73-448e-bca2-8eadb9acd32f.png)

**Instruções:**
https://developers.mercadolivre.com.br/pt_br/realizacao-de-testes

**Cartões de teste:**
https://www.mercadopago.com.br/developers/en/guides/resources/localization/local-cards

**Gerador de CPF para testes:**
https://www.4devs.com.br/gerador_de_cpf

**Aqui está um vídeo explicando como fazer isso, mas não funciona;**
https://www.youtube.com/watch?v=-6jUgwdhyQ4

**Comment by woj1965** [comment]
tente utilizar no início do nome do titular do cartão os códigos abaixo, conforme o objetivo do teste.
No meu teste de aprovação usei **APRO Test**

![image](https://user-images.githubusercontent.com/23257632/145037678-671c881e-1e8c-4fd8-80dc-cfbe7fea88ac.png)

**Reply by EnderMaldonado** [reply]
Sou burro kkk, você está certo. Só tinha que descer um pouco na pagina dos cartões para ver os nomes abaixo.
Muito Obrigado!

**Comment by usinadecoisas** [comment]
Desculpe Ender, sei que faz muito tempop que passou por aqui, mas eu como marinheiro de primeira viagem estou enfrentando uma espiral de erros para cadastrar usuários teste, acredito que é por conta de OAuth, que parece me qualificar em 70% de segurança somente, dentro do cadastro da ferramenta onde se escolhe escopos e cadastra URLs e SECRET_KEY, SECRET_ID tenho a aplicação certificada, mas nO menu de seleção ela aparece como não certificada, tem algo que me ajude a sair dessa? Antecipadamente Agradeço! : )
observação: esta ferramenta ainda está em desenvolvimento, não é necessário que esteja totalmente concluído o projeto não neh?
